CONTRIBUTING.md

```diff
@@ -125,54 +125,41 @@ make sure you run all this from a filesystem that supports this.
    git submodule update
```

Comment:
Is this step actually necessary anymore?

Reply:
sure, this is the build script